### PR TITLE
[grafana] conf(grafana): Add `folderUid` option

### LINKED ISSUE #2955: [Grafana] Add `folderUid` support for dashboards provider
We hit a problem in our deployment when we had 2 environments (production and dev) and we configured sidecar for provisioning Grafana dashboards:
```yaml
    sidecar:
      dashboards:
        enabled: true
        folder: /tmp/dashboards
        label: grafana_dashboard
        labelValue: "1"
        provider:
          allowUiUpdates: true
          folder: 'Provisioned'
          name: sidecarProvider
          orgid: 1
          type: file
          updateIntervalSeconds: 60
        resource: configmap
```
Related template code below:
https://github.com/grafana/helm-charts/blob/06500b7ed7ff1c281f083139e55d595f985c17a2/charts/grafana/templates/_config.tpl#L141-L157

The problem is when you want a provision a dashboard with the `Dashboard panel` where you define `Folder` which is represented as a UUID in JSON.  
That **UUID is for each environment different** and thus it doesn't work well in one of our deployments.

The solution is possible - add support to specify static UUID via `folderUid` in the Helm chart in that `provider` section, like `folderUid: 'ba6053fb-2100-4e8d-b601-6163c90e06fb'`.

What do you think? 🤔

### PR DESCRIPTION
**Pull request by Rohlik**
My first PR 🎉 !
This simple PR should allow users to be able to set a `folderUid` which may help solve issues in some deployments, see referencing ticket for more info.

TICKET: Fix #2955

### CODE REVIEW

**Review by zanhsieh — CHANGES_REQUESTED** [review]
@Rohlik 
Can you bump the version in Chart.yaml and sign DCO please?

**Comment by Rohlik** [comment]
@zanhsieh Thx for the review.
DCO is already signed.
Bumping `Chart.yaml` now.

File: charts/grafana/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: grafana
-version: 7.3.0
+version: 7.3.1
```
Comment (zanhsieh) [inline charts/grafana/Chart.yaml:3]:
```suggestion
version: 7.4.0
```

**Review by zanhsieh — CHANGES_REQUESTED** [review]

**Review by zanhsieh — APPROVED** [review]

**Comment by Rohlik** [comment]
@zanhsieh Hey 👋🏼, can we merge this? I do not want to do a rebase again and again 🙄.

**Comment by zanhsieh** [comment]
> @zanhsieh Hey 👋🏼, can we merge this? I do not want to do a rebase again and again 🙄.
@Rohlik 
I can't. It required two approvals. I don't control this grafana repo, grafana.com does. Long long time ago only requires single approval but they change the policy somehow.

**Review by zalegrala — APPROVED** [review]

**Review by zalegrala — APPROVED** [review]

**Comment by Rohlik** [comment]
Just you @zanhsieh and we are ready to go 🚀.

**Comment by zalegrala** [comment]
Thanks for the contribution @Rohlik.

**Pull request merged**